scripts/gulp/sh.ts

```diff
@@ -7,7 +7,7 @@ const sh = (command: string, cwd?: string, pipeOutputToResult: boolean = false):
     const options: childProcess.SpawnOptions = {
       cwd: cwd || process.cwd(),
       env: process.env,
-      stdio: pipeOutputToResult ? 'pipe' : [0, 1, 2],
+      stdio: pipeOutputToResult ? 'pipe' : 'inherit',
```

Comment:
What's the result of this setting?

Reply:
It's the same behavior, just a clearer way of writing it. There were originally going to be more changes to this file which I reverted, but this seemed worth keeping. 
https://nodejs.org/docs/latest-v12.x/api/child_process.html#child_process_options_stdio